jupyterhub/values.yaml

```diff
@@ -650,6 +650,13 @@ ingress:
   pathType: Prefix
   tls: []
 
+httpRoute:
+  enabled: false
+  hostnames: []
+  gateway:
+    name: your-gateway
+    namespace: namespace-for-your-gateway
```

Comment:
To fix the problems I had with the linting I decided to go for default values for the gateway. Hope this is okay.